src/stories/InteractiveStepSubHeader.stories.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, {useRef} from 'react';
+import {Button, View} from 'react-native';
+import InteractiveStepSubHeader from '@components/InteractiveStepSubHeader';
+
```

Comment:
Is we still have some time now between addressing final feedback coming in, could you look into migrating this one to Typescript? there are already some TS stories so might be possible, thanks